bridge/core/dom/element.cc

```diff
@@ -351,7 +374,8 @@ ScriptPromise Element::toBlob(double device_pixel_ratio, ExceptionState& excepti
 }
 
 std::string Element::outerHTML() {
-  std::string s = "<" + nodeNameLowerCase();
+  std::string tagname = local_name_.ToStdString(ctx());
```

Comment:
No need to convert to lowercase here anymore?

Reply:
name are tolowercased at createElement() function.

https://github.com/openwebf/webf/pull/126/files#diff-21f9bbf31500c133b00f2df750d9a69f2532cbd12a7eba8c9310f7bb22d2c2f0R43